cypress/support/step_definitions/api_admin_replace_api_key.js

```diff
@@ -27,41 +27,38 @@ then('the API key is updated', () => {
 	cy.get('@registryName').then((registryName) => {
 		cy.fixture('entityTestData').then((testData) => {
 			cy.get('@registryAdminApiKey').then((registryAdminApiKey) => {
-				cy.get('').then((registryApiKey) => {
-					let url = '/registry/' + registryName + '/entity'
-					cy.request({
-						url: url,
-						method: 'POST',
-						body: testData,
-						failOnStatusCode: false,
-						headers: {
-							'api-key': registryApiKey,
-							'content-type': 'application/json'
-						}
-					}).then((response) => {
-						cy.expect(response.status).to.equal(403)
-					})
+				let url = '/registry/' + registryName + '/entity'

```

Comment:
We need to have a discussion about cypress test code and what we expect of JS code style